### PR TITLE
Add console.logLocal, this circumvents sending the log to stdout/file

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

If you want to use `logLocal` make sure to call setupLocalLogs before hand.

### CODE REVIEW

File: shared/engine/index.js [hunk 1]
```diff
@@ -154,14 +157,14 @@ class Engine {
       result: (...args) => {
         if (once) {
           if (printRPC) {
-            console.log('RPC ▼ result bailing on additional calls: ', method, param, ...args)
+            console.logLocal('RPC ▼ result bailing on additional calls: ', method, param, ...args)
```
Comment (chrisnojima) [inline shared/engine/index.js]: i think we should avoid enhancing console with our custom stuff. us monkeypatching remote components to route the output i think is ok but i don't want a bunch of random methods in there that are new

Comment (chrisnojima) [inline shared/engine/index.js]: so i'd advocate import logLocal from 'loglocal'

**Comment by MarcoPolo** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
:+1:

**Pull request merged**